Rakefile

```diff
@@ -15,3 +16,13 @@ Rake::TestTask.new(:test) do |t|
   t.test_files = FileList['test/**/*_test.rb']
   t.verbose = true
 end
+
+namespace :jenkins do
+  desc nil # No description means it's not listed in rake -T
+  task :setup do
+    ENV["CI_REPORTS"] = 'jenkins/reports/unit/'
```

Comment:
Thinking more about this, I think this is something that should be done in Jenkins. There we already have the collector so the path is known.